src/Endpoint/Abstracts/AbstractSugarBeanEndpoint.php

```diff
@@ -367,6 +367,14 @@ public function relate(string $linkName, string $related_id): AbstractSugarBeanE
         return $this->link($linkName, $related_id);
     }
 
+    public function auditLog(): AbstractSugarBeanCollectionEndpoint
+    {
+        $client = $this->getClient();
+        $client->setVersion("11_12");
```

Comment:
We should technically check if the client version is greater than 11_12 before setting it. As this will override every endpoint usage after this. So we should set it, and put it back if we are changing it